fw/http_limits.c

```diff
@@ -1404,6 +1407,15 @@ frang_tls_conn_limit(FrangAcc *ra, FrangGlobCfg *conf, int hs_state)
 	switch (hs_state) {
 	case TTLS_HS_CB_FINISHED_NEW:
 		ra->history[i].tls_sess_new++;
+
+		if (conf->tls_new_conn_burst
```

Comment:
Can you explain why we move this code here?

Reply:
Just not to create second counter ("i" reused in for loop, so we need "j")